sys/include/net/gnrc/netif/conf.h

```diff
@@ -76,7 +76,7 @@ extern "C" {
  * @see         net_gnrc_netif_pktq
  */
 #ifndef CONFIG_GNRC_NETIF_PKTQ_TIMER_US
-#define CONFIG_GNRC_NETIF_PKTQ_TIMER_US       (5000U)
```

Comment:
Do you know for what reason this large number was chosen in the first place?

Reply:
I'm not sure TBH. Maybe @miri64 could give a hint?
This seems to be in the order of a full IEEE 802.15.4 frame (around 4 ms).

Reply:
Yepp, that's exactly why the value was chosen, in case the device does not report not being busy in time.

Reply:
arg but that clashes with 802.15.4g devices that have a much longer airtime :confused: 

I can of course manually adjust this, but I don't like having to tweak config values based on the radio I'm using - do we even need this if we have TX done/RX done events?